.github/workflows/schedule.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: schedule
+on:
+  schedule:
+    - cron: '23 5 * * 0'
+  workflow_dispatch:
+
+jobs:
+  schedule:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/devel/openqa/ci/containers/base:latest
```

Comment:
You can use registry.opensuse.org/home/okurz/container/containers/tumbleweed:openQA-client and then don't need to call `zypper ref` and the client install